src/screentime.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+
+#if !MONOMAC
+using UIKit;
+#else
+using AppKit;
+using UIViewController = AppKit.NSViewController;
+#endif
+
+namespace ScreenTime {
+
+	[Mac (11, 0), iOS (14, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface STScreenTimeConfiguration : NSSecureCoding {
+		[Export ("enforcesChildRestrictions")]
+		bool EnforcesChildRestrictions { get; }
+	}
+
+	[Mac (11,0), iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface STScreenTimeConfigurationObserver {
+		[Export ("initWithUpdateQueue:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (DispatchQueue updateQueue);
+
+		[Export ("startObserving")]
+		void StartObserving ();
+
+		[Export ("stopObserving")]
+		void StopObserving ();
+
+		[NullAllowed, Export ("configuration", ArgumentSemantic.Strong)]
+		STScreenTimeConfiguration Configuration { get; }
+	}
+
+	[Mac (11, 0), iOS (14, 0)]
+	[BaseType (typeof (NSObject))]
```

Comment:
`[DisableDefaultCtor]` ?
not clear (check headers) if a bindle identifier is required (or not)
In doubt, disable - easier to add later than remove :)

Reply:
```suggestion
	[BaseType (typeof (NSObject))]
	[DisableDefaultCtor]
```